manual_nav/blocks/manual_nav/form_setup_html.php

```diff
@@ -13,7 +13,7 @@
 ?>
 <div class="form-group">
     <label><?php echo t('Include Image.'); ?></label>
-    <select data-field="entry-image-select" name="displayImage" class="form-control" style="width: 60%;">
+    <select data-field="entry-image-select" name="displayImage" class="form-select" style="width: 60%;">
```

Comment:
@SashaMcr Are your changes backward compatible? I guess it won't work on v8 anymore, right?
If so, please consider using both `form-control` & `form-select`.